src/lib/Behat/PageElement/Fields/MapLocation.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\EzPlatformAdminUi\Behat\PageElement\Fields;
+
+use EzSystems\EzPlatformAdminUi\Behat\Helper\UtilityContext;
+use PHPUnit\Framework\Assert;
+
+class MapLocation extends EzFieldElement
+{
+    /** @var string Name by which Element is recognised */
+    public const ELEMENT_NAME = 'Map location';
+
+    public function __construct(UtilityContext $context, string $locator, string $label)
+    {
+        parent::__construct($context, $locator, $label);
+        $this->fields['latitude'] = '#ezrepoforms_content_edit_fieldsData_field_value_latitude';
```

Comment:
Something I'd like to check: will a Content Type work with two map fields? This ID does not look unique-per-field. (I'm not talking about this Element now, but the real app).